### PR TITLE
Demo login feature inprogress branch

### PR DESCRIPTION
**Pull request by alexyoon97**
What this PR does (required):
Singing in with hardcoded user data

Screenshots / Videos (required):
Currently not working yet, since our database is not set up
demo_button

Any information needed to test this feature (required):
Click the "Try Demo" button to be logged in as Demo User

Any issues with the current functionality (optional):
I need some feedback on functionality if it's taking the right steps.

### CODE REVIEW

File: client/src/pages/SignUp/SignUpForm/SignUpForm.tsx [hunk 1]
```diff
@@ -51,70 +51,72 @@ const SignUpForm = ({ handleSubmit }: Props): JSX.Element => {
...
+                classes: { input: classes.inputs },
+              }}
+              name="username"
+              autoComplete="username"
+              autoFocus
+              helperText={touched.username ? errors.username : ''}
+              error={touched.username && Boolean(errors.username)}
+              value={values.username}
+              onChange={handleChange}
+            />
```
Comment (moffatethan) [inline client/src/pages/SignUp/SignUpForm/SignUpForm.tsx]: We won't have usernames per the mock-ups. This isn't needed 👍🏻

**Review by moffatethan — COMMENTED** [review]
This is looking good, left you a comment. Let me know when it's ready for review

File: client/src/pages/SignUp/SignUpForm/SignUpForm.tsx [hunk 2]
```diff
@@ -51,70 +51,53 @@ const SignUpForm = ({ handleSubmit }: Props): JSX.Element => {
...
-              {isSubmitting ? <CircularProgress style={{ color: 'white' }} /> : 'Create'}
-            </Button>
-          </Box>
+            <Box textAlign="center">
+              <Button type="submit" size="large" variant="contained" color="primary" className={classes.submit}>
+                {isSubmitting ? <CircularProgress style={{ color: 'white' }} /> : 'Create'}
+              </Button>
+            </Box>
+          </form>
           <DemoButton />
```
Comment (alexyoon97) [inline client/src/pages/SignUp/SignUpForm/SignUpForm.tsx:99]: Demo button is not selected as file chaged but it should be.

**Comment by alexyoon97** [comment]
@moffatethan Thanks, I requested for review.

**Review by moffatethan — APPROVED** [review]

**Pull request merged**